Classes/Profile/ProfileComponent.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  ProfileComponent.swift
+//  Freetime
+//
+//  Created by Sash Zats on 2/3/18.
+//  Copyright © 2018 Ryan Nystrom. All rights reserved.
+//
+
+import Theodolite
+import Flexbox
+
+/**
+
+ +------------------------+
+ | +------+               |
+ | |      |  Username     |
+ | |      |  Full Name    |
+ | |      |  Bio          |
+ | +------+               |
+ +------------------------+
+
+ */
+final class ProfileComponent: Component, TypedComponent {
+  typealias PropType = (ProfileResult)
+
+  override func render() -> [Component] {
+    let options = LabelComponent.Options(
+      view: ViewOptions(backgroundColor: .white),
+      font: UIFont(name: "Courier", size: 12)!)
+    var verticalChildren: [FlexChild] = [
+      FlexChild(LabelComponent(("⚙️ \(self.props.id)", options))),
+      FlexChild(LabelComponent(("🔑 \(self.props.login)", options))),
+    ]
+    if let name = self.props.name {
+      verticalChildren.append(FlexChild(LabelComponent(("👤 \(name)", options))))
```

Comment:
Man, we gotta add a render-to-nil option for flex children. That would allow this to be done inline in a much cleaner format I think.